nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/test/resources/test-directories-sql-contrib.xml

```diff
@@ -37,6 +37,14 @@
       <createTablePolicy>on_missing_columns</createTablePolicy>
     </directory>
 
+    <extension target="org.nuxeo.ecm.directory.GenericDirectory" point="directories">
+      <directory name="anyDirectory" extends="template-directory">
```

Comment:
Regarding the review's comments in java code, this contribution should be in another module.
Furthermore, extension is not in the right location of XML (inside another extension element).